release-team/role-handbooks/release-signal/README.md

```diff
@@ -33,25 +33,24 @@
     * [Finding a Flaky Test](#finding-a-flaky-test)
     * [Priority Labels](#priority-labels)
     * [Milestones](#milestones)
-<!-- TOC -->
```

Comment:
You may remove this comment altogether; I'm not opposed to it. This was generated by the IDE.